e2e-tests/playwright.config.ts

```diff
@@ -1,54 +1,129 @@
 import { defineConfig, devices } from '@playwright/test'
+import { config } from 'dotenv'
+
+const CI = !!process.env.CI
+const { SLOW_MO, OURHEALTH_PARTICIPANT_URL } = process.env
 
 /**
- * Read environment variables from file.
- * https://github.com/motdotla/dotenv
+ * Pass in `TEST_ENV` environment variable to read environment variables from .env.* file. Example: TEST_ENV=local
+ *  - override hardcoded URL in global-setup.ts
+ *  - non-default environment variables
+ *  - specify new environment variables
  */
-// require('dotenv').config();
+const { TEST_ENV } = process.env
+if (TEST_ENV) {
+  config({ path: `.env.${TEST_ENV}`, override: true })
+} else {
+  config()
+}
```

Comment:
`.env.*` is optional. If file is not found, then any default and hardcoded env variables in global-setup and playwright.config.ts will be automatically.